requirements.txt

```diff
@@ -2,7 +2,7 @@ wheel>=0.23.0
 numpy>=1.10.2
 pymongo>=3.0
 matplotlib>=1.3.0
-scipy>=0.15
+scipy>=0.17
```

Comment:
Our common installations are at ```0.18``` already (was upgraded due to some crashes before, not sure if ```0.17``` is sufficient).